lib/Conversion/CatalystQuantumToQC/CatalystQuantumToQC.cpp

```
@@ -0,0 +1,641 @@
+ *
+ * Licensed under the MIT License
+ */
+
+#include "mlir/Conversion/CatalystQuantumToQC/CatalystQuantumToQC.h" // NOLINT(misc-include-cleaner)
+
+#include "mlir/Dialect/QC/IR/QCDialect.h"
+
+#include <Quantum/IR/QuantumDialect.h>
+#include <Quantum/IR/QuantumOps.h>
+#include <cassert>
+#include <cstddef>
```

Cppcheck (2.20.0)

[information] Include file

(missingInclude)